bioconda_utils/bioconda_utils-requirements.txt

```diff
@@ -7,21 +7,21 @@ conda-forge-pinning=2024.06.01.05.54.15
 
 # basics
 python=3.10.*
-conda=24.5.*
+conda=24.9.*
 conda-libmamba-solver=24.1.*
-conda-build=24.5.*
-conda-index=0.4.*
-conda-package-streaming=0.9.*
+conda-build=24.9.*
+conda-index=0.5.*
+conda-package-streaming=0.11.*
 mamba=1.5.*
-boa=0.17.*
+#boa=0.17.*
```

Comment:
Are we using this one? This one is currently blocked by https://github.com/conda-forge/boa-feedstock/pull/85

Reply:
Looks like the boa imports were all removed in https://github.com/bioconda/bioconda-utils/pull/980

Reply:
```suggestion
```